pandas/core/base.py

```diff
@@ -713,8 +713,15 @@ def argmax(
         --------
         Consider dataset containing cereal calories
 
-        >>> s = pd.Series({{'Corn Flakes': 100.0, 'Almond Delight': 110.0,
-        ...                'Cinnamon Toast Crunch': 120.0, 'Cocoa Puff': 110.0}})
+        >>> s = pd.Series(
+        ...     [100.0, 110.0, 120.0, 110.0],
```

Comment:
If ruff can't handle `{{`, it might be better to open an issue at ruff instead of re-writing the documentation.

Reply:
True but IMO I think it's okay to rewrite these ones for now just to avoid the potential `{{` confusion for future contributors